packages/field-plugin/helpers/vite/src/manifest.ts

```diff
@@ -48,12 +49,40 @@ const validateSchema = (manifest: Manifest): void => {
     throw new Error(`When declared, the 'options' property should be an array`)
   }
 
-  manifest.options?.forEach((option) => {
-    if (!('name' in option && 'value' in option)) {
-      throw new Error(
-        `Some of the defined 'options' are invalid. ` +
-          `Please, make sure they contain a 'name' and 'value' properties`,
+  validateOptions(manifest.options)
+  return
+}
+
+const validateOptions = (options: unknown[]): void => {
+  const incorrectValues: string[] = []
+
+  for (const option of options) {
+    if (!isOptionObject(option)) {
+      incorrectValues.push(
+        `${JSON.stringify(option)} --> Incorrect object value. Must be of type {"name": string, "value": string}.`,
       )
+      continue
     }
-  })
+
+    if (!isString(option.value)) {
+      incorrectValues.push(
+        `${JSON.stringify(option)} --> Incorrect value type. Must be of type string.`,
+      )
+      continue
+    }
+
+    if (isEmpty(option.value)) {
```

Comment:
after reviewing this, i am not sure if checking empty string value makes sense, as the user might want to have it this way. WDYT?

Reply:
Good point, Bibi 🙌 
I have used empty options in some of our ecommerce integrations (Salesforce for example) just to have them listed in the sandbox easily once it would help me during development or to sync them when using the CLI to do the deploys. 
So I wonder, may others want to use the same way? 🤔